test/unit/schemas/DeclarationStatusResponseSpec.scala

```diff
@@ -26,14 +26,14 @@ import uk.gov.hmrc.play.test.UnitSpec
 
 import scala.xml.{Elem, Node, SAXException}
 
-class StatusQueryResponseSpec extends UnitSpec with MockitoSugar with BeforeAndAfterEach {
+class DeclarationStatusResponseSpec extends UnitSpec with MockitoSugar with BeforeAndAfterEach {
 
   protected val MockConfiguration = mock[Configuration]
   protected val MockXml = mock[Node]
 
   protected val propertyName: String = "xsd.locations.statusqueryresponse"
```

Comment:
This spec has a lot of references to `StatusQueryResponse` including this line. Probably should be changed.